### ISSUE TITLE
Iris not starting

### ISSUE THREAD

**Issue by patrickhener** [jaedb/Iris#478]
*What part of Iris is your question about?*
General: Starting Iris

**Your question**
I cannot get iris to start in any way, anoymore (it was working times ago)

I have Iris and Mopidy installed, but Iris is not available under http://localhost:6680/iris

I installed iris by `pip install mopidy-iris`

My config looks like this:

```
--- ~ » sudo mopidyctl config
Running "/usr/bin/mopidy --config /usr/share/mopidy/conf.d:/etc/mopidy/mopidy.conf config" as user mopidy
[core]
cache_dir = /var/cache/mopidy
config_dir = /etc/mopidy
data_dir = /var/lib/mopidy
max_tracklist_length = 10000
restore_state = false

[logging]
verbosity = 0
format = %(levelname)-8s %(asctime)s [%(process)d:%(threadName)s] %(name)s\n  %(message)s
color = true
config_file = /etc/mopidy/logging.conf

[audio]
mixer = software
mixer_volume = 
output = pulsesink server=127.0.0.1
buffer_time = 

[proxy]
scheme = 
hostname = 
port = 
username = 
password = 

[file]
enabled = false  ; Extension disabled by user config.

[http]
enabled = true
hostname = 127.0.0.1
port = 6680
zeroconf = Mopidy HTTP server on $hostname
allowed_origins = 
csrf_protection = true
default_app = mopidy

[m3u]
enabled = false  ; Extension disabled by user config.

[softwaremixer]
enabled = true

[stream]
enabled = true
protocols = 
  http
metadata_blacklist = 
timeout = 5000

[tunein]
enabled = true
timeout = 5000
filter = 

[spotify]
enabled = true
username = patrickhener
password = ********
client_id = 0eb8409e-6f3d-412e-9979-a476af902f39
client_secret = ********
bitrate = 320
volume_normalization = true
private_session = false
timeout = 10
allow_cache = true
allow_network = true
allow_playlists = true
search_album_count = 20
search_artist_count = 10
search_track_count = 50
toplist_countries = 

[spotify_web]
enabled = false  ; Extension disabled by user config.

[mpd]
enabled = true
hostname = 127.0.0.1
port = 6600
password = 
max_connections = 20
connection_timeout = 60
zeroconf = Mopidy MPD server on $hostname
command_blacklist = 
  listall
  listallinfo
default_playlist_scheme = m3u

[local]
enabled = false  ; Extension disabled by user config.

[local-images]
enabled = false  ; Extension disabled by user config.
```

When starting mopidy I will get this in journal (starting it as a systemd service):

```
Jan 20 12:21:01 hummingbird systemd[1]: Started Mopidy music server.
Jan 20 12:21:01 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:01,899 [61144:MainThread] mopidy.__main__
Jan 20 12:21:01 hummingbird mopidy[61144]:   Enabled extensions: spotify, http, tunein, mpd, stream, softwaremixer
Jan 20 12:21:01 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:01,899 [61144:MainThread] mopidy.__main__
Jan 20 12:21:01 hummingbird mopidy[61144]:   Disabled extensions: spotify_web, m3u, local-images, local, file
Jan 20 12:21:01 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:01,987 [61144:MainThread] mopidy.commands
Jan 20 12:21:01 hummingbird mopidy[61144]:   Starting Mopidy mixer: SoftwareMixer
Jan 20 12:21:01 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:01,988 [61144:MainThread] mopidy.commands
Jan 20 12:21:01 hummingbird mopidy[61144]:   Starting Mopidy audio
Jan 20 12:21:01 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:01,989 [61144:MainThread] mopidy.commands
Jan 20 12:21:01 hummingbird mopidy[61144]:   Starting Mopidy backends: StreamBackend, TuneInBackend, SpotifyBackend
Jan 20 12:21:02 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:02,000 [61144:Audio-2] mopidy.audio.actor
Jan 20 12:21:02 hummingbird mopidy[61144]:   Audio output set to "pulsesink server=127.0.0.1"
Jan 20 12:21:02 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:02,420 [61144:SpotifyEventLoop] mopidy_spotify.backend
Jan 20 12:21:02 hummingbird mopidy[61144]:   Logged in to Spotify in online mode
Jan 20 12:21:02 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:02,926 [61144:SpotifyBackend-5] mopidy_spotify.web
Jan 20 12:21:02 hummingbird mopidy[61144]:   Logged into Spotify Web API as patrickhener
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,543 [61144:SpotifyBackend-5] mopidy_spotify.playlists
Jan 20 12:21:05 hummingbird mopidy[61144]:   Refreshed 11 Spotify playlists
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,544 [61144:MainThread] mopidy.commands
Jan 20 12:21:05 hummingbird mopidy[61144]:   Starting Mopidy core
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,552 [61144:MainThread] mopidy.commands
Jan 20 12:21:05 hummingbird mopidy[61144]:   Starting Mopidy frontends: HttpFrontend, MpdFrontend
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,553 [61144:HttpFrontend-10] mopidy.http.actor
Jan 20 12:21:05 hummingbird mopidy[61144]:   HTTP server running at [::ffff:127.0.0.1]:6680
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,556 [61144:MainThread] mopidy_mpd.actor
Jan 20 12:21:05 hummingbird mopidy[61144]:   MPD server running at [::ffff:127.0.0.1]:6600
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,558 [61144:MainThread] mopidy.commands
Jan 20 12:21:05 hummingbird mopidy[61144]:   Starting GLib mainloop
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,911 [61144:MpdSession-12] mopidy_mpd.session
Jan 20 12:21:05 hummingbird mopidy[61144]:   New MPD connection from [::ffff:127.0.0.1]:40912
Jan 20 12:21:05 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:05,994 [61144:MpdSession-13] mopidy_mpd.session
Jan 20 12:21:05 hummingbird mopidy[61144]:   New MPD connection from [::ffff:127.0.0.1]:40914
Jan 20 12:21:06 hummingbird mopidy[61144]: INFO     2020-01-20 12:21:06,027 [61144:MpdSession-14] mopidy_mpd.session
Jan 20 12:21:06 hummingbird mopidy[61144]:   New MPD connection from [::ffff:127.0.0.1]:40916
```

Here are my version information:

```
--- ~ » mopidy --version
Mopidy 3.0.1
--- python3.8/site-packages » ll | grep -i iris
drwxr-xr-x      - root  9 Jan 10:32  Mopidy_Iris-3.43.0-py3.8.egg
```

So how can I solve this? Thanks in advance

**Comment by kingosticks** [comment]
Intentionally or not, you have upgraded to Mopidy v3. Iris is not yet compatibe with Mopidy 3. See https://github.com/jaedb/Iris/issues/356. In the meantime, if you might want to downgrade to Mopidy v2.3:
```
sudo apt install -y --allow-downgrades mopidy=2.3.1-1
sudo apt-mark hold mopidy
```

Also, please use `sudo mopidyctl deps` to show version information.

**Comment by patrickhener** [comment]
Well it is not that easy I figured. As I am on Arch Linux I cannot just downgrade mopidy. Mopidy Iris will need to install mopidy-local (3.xx) as a requirement which itself requires mopidy to be in version 3. something. So there is no setup in which I might be able to run mopidy-iris right now. I guess I need to wait for iris to be ported to support python3 then..

**Comment by jaedb** [comment]
The wait won't be too long @patrickhener. I'm preparing the Docker images as we speak and should have something released in the next few days.

**Comment by patrickhener** [comment]
Thanks alot for the work buddy

**Comment by jaedb** [comment]
Docker image is now released, and Iris `3.44` is on PyPi.
